content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/README.md

```diff
@@ -40,6 +40,8 @@ device) is disabled.
 4. `./lazyThreshold` - defines the number of pixel an image is getting loaded before it gets visible and lazy loading is enabled. 
 Default is set to 0.
 5.  `./enableDmFeatures` - if `true`, Dynamic Media features are enabled.
+6. `./directLoadFromDAM` - defines if DAM assets are addressed directly using the DAM asset URL
```

Comment:
This reads misleading, we're still loading the images through the AIS. Wondering if we should just use the DAM renditions instead and skip AIS processing completely for these files.

Reply:
Perhaps change to:
> `./damBasedURLs` - defines if image URLs are based on the DAM asset path